src/queries/api.js

```diff
@@ -395,6 +401,17 @@ const api = {
           }),
     };
   },
+  user() {
```

Comment:
I don't think we actually need this new API call. We can get this data from the Knack object which you can get from the global window...

`Knack.user.attributes.values` returns all the fields you need:

![Screen Shot 2019-08-12 at 3 33 37 PM](https://user-images.githubusercontent.com/5697474/62897252-7ddcd500-bd18-11e9-9b27-5b00e44c5cd4.png)

Reply:
@mateoclarke Can we go over this together? I'm not seeing the same data available on my end.

Reply:
Mateo and I looked over this, and it looks like the Knack object no longer exposes this data, so... sticking with the API call.